projects/exui-common-lib/src/lib/components/find-person/find-person.component.html

```diff
@@ -1,12 +1,12 @@
-<h1 class="govuk-heading-l">
+<h1 class="govuk-heading-l" *ngIf="title && title.length">
   <span class="govuk-caption-l">{{title}}</span>
   {{boldTitle}}
 </h1>
 <div class="govuk-form-group" [formGroup]="findPersonGroup"
-     [ngClass]="{'form-group-error': !findPersonGroup.valid}">
-  <fieldset class="govuk-fieldset" aria-describedby="">
-    <div id="waste-hint" class="govuk-hint">
-      Type the name of the person and select them.
+     [ngClass]="{'form-group-error': !findPersonGroup.valid && submitted}">
+  <fieldset class="govuk-fieldset" aria-describedby="inputSelectPerson">
+    <div class="govuk-hint" *ngIf="subTitle && subTitle.length">
+      {{subTitle}}
```

Comment:
<fieldset class="govuk-fieldset" aria-describedby="sub-title-hint">
    <div id="sub-title-hint" class="govuk-hint" *ngIf="subTitle && subTitle.length">
      {{subTitle}}

Reply:
Correct me if I am wrong but I thought the `fieldset` is describing the `input` not the `subTitle`.

Reply:
The subTitle here is 'Type the name of the person and select them.'
The inputSelectPerson has got the 'aria-label="select a person"'.
In you case the screen reeder might read twice for "select a person".
We expect the screen reader read 'Type the name of the person and select them.' if it's existing first, then read 'select a person'.

Reply:
Makes sense thanks for clarifying.

Reply:
Committed your suggested changes 😊